hystrix/metrics.go

```diff
@@ -95,6 +95,9 @@ func (m *metricExchange) IncrementMetrics(wg *sync.WaitGroup, collector metricCo
 		collector.IncrementAttempts()
 		collector.IncrementErrors()
 	}
+	if update.Types[0] == "queued" {
+		collector.IncrementQueueSize()
+	}
```

Comment:
as described in my comment in `hystrix.go`, this won't work to accurately track the event rate. a types list of `[ queued, failure, fallback-success ]` should apply all of:

```go
collector.IncrementQueueSize()
collector.IncrementFailures()
collector.IncrementAttempts()
collector.IncrementErrors()
collector.IncrementFallbackSuccesses()
```